channels-rke2.yaml

```diff
@@ -303,3 +315,15 @@ releases:
       rke2-canal:
         repo: rancher-rke2-charts
         version: v3.20.3-build2022011406
+  - version: v1.22.7-rc2+rke2r1
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-22-7-rc2-rke2r1
+      <<: *serverArgs-v1-22-6-rke2r1
+    agentArgs: &agentArgs-v1-22-7-rc2-rke2r1
+      <<: *agentArgs-v1-22-6-rke2r1
+    charts: &charts-v1-22-7-rc2-rke2r1
+      <<: *charts-v1-22-6-rke2r1
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.20.3-build2022011406
```

Comment:
This appears to be the same chart version as in `charts-v1-22-6-rke2r1`; I think you can just use that anchor without any changes or even creating a new anchor.